spec/request/api/v1/auth/sign_in_spec.rb

```diff
@@ -0,0 +1,90 @@
+require 'rails_helper'
+require 'faker'
+
+describe 'Auth', type: :request do
+  let(:user) { build(:user) }
```

Comment:
You should create the user to later test the login:
`let!(:user) { create(:user, email: 'test@example.com', password: 'password') }`